src/zulip.ts

```diff
@@ -15,17 +21,25 @@ type Profile = {
 	presence: Presence;
 };
 
+const botMessage = ({ name, meetUrl }: { name: string; meetUrl: string }) => `
+Hello friends, I'm ${name}, your friendly neighborhood open office bot. My purpose is to facilitate async team bonding by providing tools for working in the open.
+
+Visit your peers at ${meetUrl}, or ask what I can do by pinging or PM-ing me with "\`help\`".
```

Comment:
There's no specific response for help yet, but we can add it later

Reply:
Are we still looking at all of this being split into the three places? Threads, user, and in conversations?

Reply:
Yes! The user will display number of people, the thread is for the link and introducing the purpose of the bot, and the private message for now just sends the link but will do the "help" functionality at a future date.

Reply:
We updated it so that a help ping to the bot will send the welcome message ("Hello friends...")